### DISCUSSION TITLE
Features of ExplorerPatcher not functioning after latest update of Windows 11

### DISCUSSION THREAD

**Discussion by PeculiarReality** [valinet/ExplorerPatcher#702] [Q&A]
(Using 7+ Taskbar Tweaker and ClassicShell, but issues persist with 7+ and ClassicShell uninstalled. Everything works fine in the Win11 version of the taskbar.)
(Latest commit/prerelease version. Issues persisted on latest release.)

Since the latest update of Windows 11 (Build 22000.co_release.210604-1628, Windows 11 Home), 10-style taskbar functions do not seem to work properly. Clock does not display at all, icons take up the entire taskbar despite being set to group, and the right-click context menu for the taskbar is empty. Buttons in the context menu function properly but have no text.

Example image provided below.

![image](https://user-images.githubusercontent.com/11938309/150435343-e9ad779f-59e4-476f-ba06-25343a6d7fb3.png)

**Comment by valinet** [comment]
What Windows build exactly?

**Reply by PeculiarReality** [reply]
As mentioned in the post, build 22000.co_release.210604-1628, Windows 11 Home. I got this information from the 'Show Windows Build Information on the Desktop' feature.

**Reply by valinet** [reply]
Open `winver` and tell what it says there. Is it 22000.434 or something like that?

**Reply by PeculiarReality** [reply]
OS Build 22000.434

**Reply by valinet** [reply]
Okay, I am using that and it is working fine. I don't know what could be causing the issues. Maybe try the latest pre-release of EP...?

**Reply by PeculiarReality** [reply]
I am using the latest pre-release currently, as far as I can tell. Is there a way to uninstall ExplorerPatcher and reinstall it?

**Reply by valinet** [reply]
Yeah, uninstall and install it again. Which means run `ep_setup.exe` twice. But I don't think it will help much.

**Reply by PeculiarReality** [reply]
Running ep_setup.exe only seems to reinstall it over the existing build.

**Reply by valinet** [reply]
And what were you expecting? It reinstalls it, as you asked about.

**Reply by PeculiarReality** [reply]
Yes, but I was wondering if there was a way to _uninstall_ it so I could reinstall it from ep_setup.exe,

**Reply by PeculiarReality** [reply]
Messing around with the settings, I just figured out that the issue is directly related to the "Enable advanced mitigations for correcting rendering using classic theme" setting... which I genuinely don't remember enabling. At all.

**Reply by valinet** [reply]
Yeah, indeed, that can cause issues. That's only useful when using the classic theme, it breaks things in any other scenario, as you found out. Good thing that you sorted it out.
